aries/aries_vcx_core/Cargo.toml

```diff
@@ -13,8 +13,12 @@ vdr_proxy_ledger = ["credx", "dep:indy-vdr-proxy-client"]
 # Feature flag to allow legacy proof verification
 legacy_proof = []
 
+askar_wallet = ["dep:aries-askar"]
+
 [dependencies]
 agency_client = { path = "../misc/legacy/agency_client" }
+aries-askar = { git = "https://github.com/hyperledger/aries-askar.git", tag = "v0.3.0", optional = true }
+bs58 = { version = "0.5" }
```

Comment:
Since these deps (`bs58`, `sodiumoxide`) were added as part of askar impl, they should be also optional.

Reply:
Updated.